src/llm/legacy/servable_initializer.cpp

```diff
@@ -0,0 +1,130 @@
+//*****************************************************************************
+// Copyright 2025 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <openvino/genai/continuous_batching_pipeline.hpp>
+#include <openvino/openvino.hpp>
+#include <spdlog/spdlog.h>
+
+#pragma warning(push)
+#pragma warning(disable : 4005 4309 6001 6385 6386 6326 6011 4005 4456 6246)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_graph.h"
+#pragma GCC diagnostic pop
+#pragma warning(pop)
+
+#include "../../json_parser.hpp"
+#include "../../logging.hpp"
+#include "../../mediapipe_internal/mediapipe_utils.hpp"
+#include "../../status.hpp"
+#include "servable.hpp"
+#include "servable_initializer.hpp"
+
+namespace ovms {
+
+ov::genai::SchedulerConfig LegacyServableInitializer::prepareDraftPipelineSchedulerConfigExperimental(const mediapipe::LLMCalculatorOptions_PipelineConfig& draftPipelineConfig) {
+    ov::genai::SchedulerConfig config;
+    config.max_num_batched_tokens = draftPipelineConfig.max_num_batched_tokens();
+    config.cache_size = draftPipelineConfig.cache_size();
+    config.dynamic_split_fuse = draftPipelineConfig.dynamic_split_fuse();
+    config.max_num_seqs = draftPipelineConfig.max_num_seqs();
+    config.enable_prefix_caching = draftPipelineConfig.enable_prefix_caching();
+    return config;
+}
+
+ov::genai::SchedulerConfig LegacyServableInitializer::prepareDraftPipelineSchedulerConfig(const mediapipe::LLMCalculatorOptions& nodeOptions) {
+    ov::genai::SchedulerConfig config;
+    config.max_num_batched_tokens = nodeOptions.has_draft_max_num_batched_tokens() ? nodeOptions.draft_max_num_batched_tokens() : nodeOptions.max_num_batched_tokens();
+    config.cache_size = nodeOptions.has_draft_cache_size() ? nodeOptions.draft_cache_size() : nodeOptions.cache_size();
+    config.dynamic_split_fuse = nodeOptions.has_draft_dynamic_split_fuse() ? nodeOptions.draft_dynamic_split_fuse() : nodeOptions.dynamic_split_fuse();
+    config.max_num_seqs = nodeOptions.has_draft_max_num_seqs() ? nodeOptions.draft_max_num_seqs() : nodeOptions.max_num_seqs();
+    config.enable_prefix_caching = nodeOptions.enable_prefix_caching();
+    return config;
+}
```

Comment:
This is required if servable supports speculative decoding. Perhaps we could set it to false and comment this out for now until tests are enabled?